vertx-opentelemetry/src/main/asciidoc/index.adoc

```diff
@@ -0,0 +1,72 @@
+= Vertx OpenTelemetry
+
+Vert.x integrates with OpenTelemetry thanks to the Jaeger client.
```

Comment:
This seems a copy-past of https://github.com/eclipse-vertx/vertx-tracing/blob/master/vertx-opentracing/src/main/asciidoc/index.adoc, and it doesn't feel quite right.
```suggestion
Vert.x integrates with OpenTelemetry.
```

Reply:
:hand_over_mouth: